RestrictEvents/RestrictEvents.cpp

```diff
@@ -263,6 +260,55 @@ struct RestrictEventsPolicy {
 		return true;
 	}
 
+	static void getBlockedProcesses(BaseDeviceInfo *info) {
+		// Updates procBlacklist with list of processes to block
+		char duip[128] { "auto" };
+		if (PE_parse_boot_argn("revblock", duip, sizeof(duip))) {
+			DBGLOG("rev", "read revblock from boot-args");
+		} else if (readNvramVariable(NVRAM_PREFIX(LILU_VENDOR_GUID, "revblock"), u"revblock", &EfiRuntimeServices::LiluVendorGuid, duip, sizeof(duip))) {
+			DBGLOG("rev", "read revblock from NVRAM");
+		}
+
+		char *value = reinterpret_cast<char *>(&duip[0]);
+		value[strlen(duip)] = '\0';
```

Comment:
This can do out of bounds. Should instead pass 1 byte less to variable getter as PE_parse_boot_argn is guaranteed to \0-terminate, and initialising with {"auto"} is guaranteed to fill all zeroes up to the end of the array.